packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -96,6 +101,18 @@ export default decorate([
             )
           )
           success(_('hubImportNotificationTitle'), _('successfulInstall'))
+          const olderTemplates = filter(templates, template =>
```

Comment:
What if one template installation fails and other installations succeed?

Reply:
So do you suggest that we don't execute them in parallel ?

Reply:
I suggest to delete the resource after a success installation.